cmake/ctest/drivers/geminga/TrilinosCTestDriverCore.geminga.gcc.cmake

```diff
@@ -71,6 +71,8 @@ MACRO(TRILINOS_SYSTEM_SPECIFIC_CTEST_DRIVER)
     SET(BUILD_DIR_NAME ${COMM_TYPE}-${BUILD_TYPE}_${BUILD_NAME_DETAILS})
   ENDIF()
 
+  SET(Trilinos_REPOSITORY_LOCATION_DEFAULT "git@github.com:muelu/Trilinos.git")
```

Comment:
Wait, hold on.  Testing pulls from two different locations.  `Nightly` pulls from a synchronized branch (develop from muelu/Trilinos in this case).  `Experimental` and `Continuous` pull from develop branch on `trilinos/Trilinos`.

So should Trilinos_REPOSITORY_LOCATION_DEFAULT point to `trilinos/Trilinos.git`?

Reply:
Why do we want to pull from `trilinos/Trilinos`? Wouldn't it make sense to have all builds build the same SHA?

Reply:
Well, I guess that change isn't really necessary. Let's only point nightly to the muelu repo.